auto_dev/constants.py

```diff
@@ -8,6 +8,8 @@
 from aea.configurations.data_types import PublicId
 
 
+THIS_REPO_ROOT = Path(__file__).parent.parent
```

Comment:
This is useful because when working with isolated filesystems, functions like `get_repo_root` correctly point to the newly scaffolded repo. But there are cases where we still need access to the `adev` repo, e.g., to fetch the current `pyproject.toml` for generating a new repo. This allows us to reach back into `adev` even when the test context is scoped to the isolated filesystem.